### PR TITLE
[MERGE AFTER pull #23 ]Feature/arduino sensors

### PR DESCRIPTION
**Pull request by IuliaUngur**

### CODE REVIEW

File: app/controllers/predictions_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class PredictionsController < ApplicationController
+  include ApplicationHelper
```
Comment (asarbu) [inline app/controllers/predictions_controller.rb]: why is this needed ? from what I know helper methods should be used in views

File: app/services/ap/version_space.rb [hunk 2]
```diff
@@ -3,11 +3,151 @@ class VersionSpace
     # valid results: plant survives, plant dies, uncertain outcome
```
Comment (asarbu) [inline app/services/ap/version_space.rb]: I feel like this service is huuuge. Some time you should refactor it.

Comment (IuliaUngur) [inline app/services/ap/version_space.rb]: it is being refactored in #29 :)

File: app/controllers/predictions_controller.rb (hunk 1 above)
Comment (IuliaUngur) [inline app/controllers/predictions_controller.rb]: the helper contains this 2 methods:
```
module ApplicationHelper
  def open(url)
    Net::HTTP.get(URI.parse(url))
  end

  def json_read(path)
    open(request.base_url + path + '.json')
  end
end
```
which I need for the JSON read from file (I wanted to read the files from URL paths if I wanted to extend this feature in the future - ie. wifi module)
and since I needed it both in views and in controller, I figured out that it would be ok to place the code here to avoid duplication
